app/table/cells/RouterLinkCell.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+
+import { useMemo } from 'react'
+
+import { useApiQuery } from '~/api'
+import { useVpcSelector } from '~/hooks'
+import { Badge } from '~/ui/lib/Badge'
+import { pb } from '~/util/path-builder'
+
+import { EmptyCell, SkeletonCell } from './EmptyCell'
+import { LinkCell } from './LinkCell'
+
+export const RouterLinkCell = ({ value }: { value?: string }) => {
+  const { project, vpc } = useVpcSelector()
+  const { data: subnet, isError } = useApiQuery('vpcRouterView', {
+    path: { router: value || '' },
+    query: { project, vpc },
+  })
```

Comment:
how to handle the non-existing router

https://github.com/oxidecomputer/console/blob/8dcddcef62b8d10dfcd3adb470439212b23b3d5e/app/table/cells/InstanceLinkCell.tsx#L19-L23